Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2018,13 +2018,13 @@ extension BrowserViewController {
         // Condition: Update from leanplum server
         // Get the A/B test variant from leanplum server
         // and update onboarding user reasearch
-        LeanPlumClient.shared.onStartLPVariable = {(lpVariable) -> () in
-            let lpVariableValue = "\(String(describing: lpVariable?.boolValue()))"
-            Sentry.shared.send(message: "Onboarding Research: onStartLPVariable - LP State - \(LeanPlumClient.shared.lpState.rawValue) | Condition: Received update from LP server with variable value as - \(lpVariableValue)", tag: .leanplum, severity: .debug, description: "Condition: Received update from LP server with variable value as - \(lpVariableValue)")
-            LeanPlumClient.shared.onStartLPVariable = nil
-            print("lp Variable from server \(lpVariableValue)")
+        LeanPlumClient.shared.finishedStartingLeanplum = {
+            let lpVariableValue = LPVariables.showOnboardingScreenAB?.boolValue()
```

Comment:
nit: `let showScreenB = LPVariables.showABOnboardingScreenB.boolValue()` so that true clearly means `b` and the var it is assigned to is more clearly named